client/components/resizable-iframe/index.jsx

```diff
@@ -11,6 +10,10 @@ const debug = debugFactory( 'calypso:resizable-iframe' );
 const noop = () => {};
 
 export default class extends Component {
+	constructor( props ) {
+		super( props );
+		this.iframeRef = createRef();
+	}
```

Comment:
Assign as an instance field, just as `state` is initialized in the same component.